tasks/FileUploadTask.md

```diff
@@ -30,6 +30,8 @@ This task aims to provide a fluent and easy to use mechanism for the consumer to
 - The task should be agnostic to the kind of upload being performed so as to support for various fileUpload scenarios e.g. **DriveItem** and **FileAttachment**. An example of the agnostic nature of task is how the task is marked as completed considering different response formats from each API:
   - If the response status is a 201.
   - An additional case for OneDrive is, if the response status is a 200 and the response contains an "id" then the task is complete.
+- MUST NOT compress the individual slices when it uploads them. This is because compressing slices with content-encoding header would this leads to invalid ranges/content length headers. (e.g. for a 100 bytes file of 10*10 bytes uncompressed slices, the content length would be ~7 bytes)
```

Comment:
```suggestion
- MUST NOT compress the individual slices when it uploads them. This is because compressing slices with content-encoding header leads to invalid ranges/content length headers. (e.g. for a 100 bytes file of 10*10 bytes uncompressed slices, the content length would be ~7 bytes)
```